spec/support/helpers/api_helper.rb

```diff
@@ -56,6 +56,9 @@ def stub_offender(offender)
 
     stub_request(:get, "#{Rails.configuration.complexity_api_host}/v1/complexity-of-need/offender-no/#{offender_no}").
       to_return(body: { level: offender.fetch(:complexityLevel) }.to_json)
+
+    stub_request(:post, "#{Rails.configuration.complexity_api_host}/v1/complexity-of-need/offender-no/#{offender_no}").
+      to_return(status: 200, body: { level: offender.fetch(:complexityLevel) }.to_json, headers: {})
```

Comment:
This isn't needed - it is a copy of the line above it with some of the defaults explictly added